lib/opal/builder/scheduler.rb

```diff
@@ -15,7 +15,7 @@ def initialize(builder)
 
   singleton_class.attr_accessor :builder_scheduler
 
-  if RUBY_ENGINE != 'opal'
+  if RUBY_ENGINE == 'ruby'
     # Windows has a faulty `fork`.
     if OS.windows? || ENV['OPAL_PREFORK_DISABLE']
```

Comment:
And perhaps move the check here. Or we can do `Kernel.respond_to? :fork` check (a little known fact about Ruby is that `respond_to?` returns false if a method is present but unimplemented on a given platform)